style.css

```diff
@@ -0,0 +1,362 @@
+  margin: 0;
+  padding: 0;
+  flex-direction: column;
+  justify-content: flex-start;
+  border: 1px solid #d0d9d4;
+  border-radius: 8px;
+  background: #ebf0ee;
+}
+
+.project .img-container {
+  flex: 1 1 252px;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.project ul {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  justify-content: space-evenly;
+}
+
+.habilities li {
+  font-size: 12px;
+  color: #36b37e;
+  font-weight: 600;
+  padding: 12px;
+  border-radius: 8px;
+  margin: 0 8px;
+  margin-bottom: 8px;
+}
+
+.project ul li {
+  background-color: #ebf0ee;
+  padding: 8px 12px;
+  margin: 2px 4px;
+  border-radius: 4px;
+  align-self: center;
+  font-size: 12px;
+  color: #3a4a42;
+  font-weight: 600;
+}
+
+.project h2 {
+  text-align: center;
+  font-size: 24px;
+  font-weight: 600;
+  color: #3a4a42;
+  line-height: 32px;
+  margin-top: 16px;
+}
+
+.project .img-footer-container {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  align-content: flex-end;
+  background: #fff;
+  border-radius: 8px 8px 6px 6px;
+}
+
+button {
+  background-color: #36b37e;
+  color: #fff;
+  font-size: 17px;
+  font-weight: 500;
+  border: none;
+  border-radius: 4px;
+  padding: 12px;
+}
+
+button:hover {
+  box-shadow: 0 8px 16px color(rgb 54 179 127 / 0.24);
+}
+
+button:active {
+  background-color: #008552;
+}
+
+.project button {
+  width: 122px;
+  height: 48px;
+  align-self: center;
+  margin-top: 29px;
+  margin-bottom: 24px;
+}
+
+.project1 {
+  grid-row: 1/1;
+}
+
+.project2 {
+  grid-row: 2/2;
+}
+
+.project3 {
+  grid-row: 3/3;
+}
+
+.project4 {
+  grid-row: 4/4;
+}
+
+.project5 {
+  grid-row: 5/5;
+}
+
+.project6 {
+  grid-row: 6/6;
+}
+
+.about-me {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: auto;
+}
+
+.about-me h1 {
+  font-size: 40px;
+  margin-top: 68px;
+  font-weight: 700;
+  color: #172b4d;
+  text-align: center;
+}
+
+.about-paragraph {
+  margin-top: 24px;
+  margin-left: 12px;
+  margin-right: 12px;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: center;
+}
+
+.habilities {
+  margin: 0;
+  margin-top: 59px;
+  padding: 0;
+  display: grid;
+  grid-template-columns: 375px;
+  grid-template-rows: repeat(3, 352px);
+  row-gap: 24px;
+}
+
+.habilities > div {
+  border: 1px solid #c1c7d0;
+  border-radius: 8px;
+  background-color: #ebf0ee;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 24px;
+}
+
+.languages {
+  grid-row: 1/1;
+}
+
+.frameworks {
+  grid-row: 2 / 2;
+}
+
+.skills {
+  grid-row: 3/3;
+}
+
+.habilities h2 {
+  padding: 0;
+  margin: 0;
+  font-size: 24px;
+  font-weight: 600;
+  color: #344563;
+}
+
+.habilities ul {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 24px;
+}
+
+.habilities img {
+  padding: 16px;
+  margin: 0;
+  height: 48px;
+  width: 48px;
```

Comment:
Please make give a good name to your id section habilities is not a unique name. Id names are very important for your practice and it makes your practice more professional.

Reply:
very well done on adding this inline comment in the correct block of code 👌 but I think they are using it as a unique id I don't see any other elements with the same id , otherwise you can mention the typo habilities -> abilities to make there work look professional.

Reply:
Yes, I have sent a message to the typo habilities -> abilities to make their work look professional and happy coding.